appOPHD/UI/FileIo.cpp

```diff
@@ -197,7 +197,7 @@ void FileIo::onFileIo()
 {
 	if(mMode == FileOperation::Save)
 	{
-		mSaveSignal(mFileName.text());
+		mFileSaveDelegate(mFileName.text());
```

Comment:
This should probably be checked before calling. The save handler is optional, so a call when it's unset will cause a seg fault.